setup.cfg

```diff
@@ -18,6 +18,7 @@ include_package_data = true
 python_requires = >=3.8
 install_requires =
 	more_itertools
+	typing_extensions; python_version < "3.11"
```

Comment:
This pull request confuses me. The type annotations are added as stubs, which means that they aren't used at runtime at all -- only by e.g. mypy. Why is a runtime dependency added here?

This forces an additional module to be installed when installing jaraco.functools.

Reply:
I suspect the dependency was necessary when the hints were inline. Feel free to send a PR to remove. If tests pass, it’s good to go

Reply:
Done: #25